frontend/src/data/nsf23306_4_2/README.md

```diff
@@ -0,0 +1,30 @@
+# Median annual salaries of full-time employed college graduates, by sex, major occupation, age, ethnicity, race, disability status, and citizenship status: 2021
+
+Table 4-2 @ Source URI [https://ncses.nsf.gov/surveys/national-survey-college-graduates/2021#data](https://ncses.nsf.gov/surveys/national-survey-college-graduates/2021#data)
+
+## Files
+
+`dcids.ts`: list of DCIDs that are available for this Data Commons dataset.
+
+`categories.ts`: category to respective dimensions mapping in this dataset.
+
+`query.ts`: exports a class named `Query_nsf23306` that enables querying the dataset with one or more `Query` objects.
+
+
```

Comment:
Can you add a comment here that they're incompatible because data is not reported at this intersection?

This will be an interesting UX challenge. I suspect we may exclude some categories as filter options if they don't exist with race/gender intersects.

Reply:
Yes. If we check the original data, disability status and citizenship status are columns, alongside ethnicity. 

![columns](https://github.com/Spelman-College/spelman-dashboard/assets/51137263/9f9d3120-e721-47bf-8e58-e7af1a05e117)

Reply:
Yes, can you add a comment to that effect to the code so it's clear for maintainers? Thanks!

Reply:
Yes. I also fixed the README and added tests- we can intersect gender with both citizenship and disbility status.